Update XML code style config in IntelliJ IDEA IDE

jacek-poreda-sonarsource opened the pull request.

julienlancelot commented:
It seems that your change doesn't allow usage of the "Reformat Code" function of Intellij, which is what I thought it would allow.

For instance, when you take this piece of code from ComponentMapper.xml :  
<img width="648" alt="Before" src="https://user-images.githubusercontent.com/370769/61389984-2fbfe780-a8ba-11e9-9fd7-7078b581d991.png">
It will be formatted like that : 
<img width="624" alt="After" src="https://user-images.githubusercontent.com/370769/61389987-32bad800-a8ba-11e9-8c29-47cba25806dd.png">

jacek-poreda-sonarsource commented:
@julienlancelot Good catch, I have fixed that issue by setting `Keep white spaces` option. Also in order to be consistent with Java eclipse formatter I've added `Hard wrap at 140`.
